.github/workflows/JuliaNightly.yml

```
@@ -0,0 +1,32 @@
+name: JuliaNightly
+# Nightly Scheduled Julia Nightly Run
+on:
+  schedule:
+    - cron: '0 2 * * *'  # Daily at 2 AM UTC (8 PM CST)
+jobs:
+  test:
+    name: Julia Nightly - Ubuntu - x64
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: nightly
+          arch: x64
+      - uses: actions/cache@v2
+        env:
+          cache-name: julia-nightly-cache-artifacts
+        with:
+          path: ~/.julia/artifacts
+          key: ${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
+          restore-keys: |
+            ${{ env.cache-name }}-
+      - uses: julia-actions/julia-buildpkg@latest
+      - run: |
+          git config --global user.name Tester
+          git config --global user.email te@st.er
```

I don't think these are needed?
```suggestion
      - run: |
          git config --global user.name Tester
          git config --global user.email te@st.er
```